File: Octopus.Client/Csharp/Feeds/GetAllFeeds.cs
// If using .net Core, be sure to add the NuGet package of System.Security.Permissions

// Reference Octopus.Client
//#r "path\to\Octopus.Client.dll"

using Octopus.Client;
using Octopus.Client.Model;

// Declare working varibles
var octopusURL = "https://youroctourl";
var octopusAPIKey = "API-YOURAPIKEY";

string spaceName = "Default";

// Create repository object
var endpoint = new OctopusServerEndpoint(octopusURL, octopusAPIKey);
var repository = new OctopusRepository(endpoint);
var client = new OctopusClient(endpoint);

try
{
    // Get space
    var space = repository.Spaces.FindByName(spaceName);
    var repositoryForSpace = client.ForSpace(space);

    // Get All feeds
    var feeds = repositoryForSpace.Feeds.FindAll();

    foreach (var feed in feeds)
    {
        Console.WriteLine("Feed Id: {0}", feed.Id);
        Console.WriteLine("Feed Name: {0}", feed.Name);
        Console.WriteLine("Feed Type: {0}", feed.FeedType);
        Console.WriteLine();
    }
}
catch (Exception ex)
{
    Console.WriteLine(ex.Message);
    return;
}

File: Octopus.Client/Go/Feeds/get_all_feeds.go
package main

import (
	"fmt"

	"github.com/OctopusDeploy/go-octopusdeploy/client"
)

var (
	// Declare working variables
	octopusURL    string = "https://youroctourl"
	octopusAPIKey string = "API-YOURAPIKEY"

	spaceName string = "Default"
)

func main() {
	client, err := client.NewClient(nil, octopusURL, octopusAPIKey, spaceName)

	if err != nil {
		// TODO: handle error
	}

	// Get all Feeds
	feeds, err := client.Feeds.GetAll()

	if err != nil {
		// TODO: handle error
	}

	for _, feed := range feeds {
		fmt.Sprintln("Feed ID: %s", feed.ID)
		fmt.Sprintln("Feed Name: %s", feed.Name)
		fmt.Sprintln("Feed Type: %s", feed.FeedType)
		fmt.Println()
	}
}
